src/Controls/src/Core/SetterSpecificity.cs

```diff
@@ -66,8 +66,12 @@ public SetterSpecificity(int style, int id, int @class, int type) : this(0, 0, 0
 
 		public int CompareTo(SetterSpecificity other)
 		{
-			//VSM setters somehow supersedes Styles
-			if (Vsm != other.Vsm)
+			//VSM setters win over Manual value, except for implicit style VSMs
+			if (Vsm != other.Vsm && (
+				   Style != 0 && Style <= StyleImplicit + 99 && other.Manual <= 0
```

Comment:
What is the significance of these number values?  Is there any documentation or explanation for what these mean?

Reply:
comments on line 33 and following